src/JavaCalculator.java

```diff
@@ -9,6 +10,9 @@ public static void main(String[] args) {
                 UI uiCalculator = new UI();
                 uiCalculator.init();
 
+                //Exception handler for AWT-EventQueue errors
+                Toolkit.getDefaultToolkit().getSystemEventQueue().push(new EventQueueExceptionHandler());
```

Comment:
What is this for?

Reply:
To handle the AWT EventQueue error we have been getting (essentially so the lecturer doesn’t mark us down for it handling it)